processor/processor.go

```diff
@@ -0,0 +1,83 @@
+package processor
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/gonative-cc/relayer/dal"
+	"github.com/gonative-cc/relayer/ika"
+)
+
+// NativeTxData represents the data extracted from a Native chain transaction.
+type NativeTxData struct {
+	TxID           uint64   `json:"tx_id"`
+	DWalletCapID   string   `json:"dwallet_cap_id"`
+	SignMessagesID string   `json:"sign_messages_id"`
+	Messages       [][]byte `json:"messages"`
```

Comment:
Maybe we can alias type  message = []byte

Reply:
Done